src/cast_core/src/Loader.cpp

```diff
@@ -77,11 +77,6 @@ struct Loader::PhaseConfig {
         };
 
         if (multipleThreadsPerCollection) {
-            if (context["Threads"]) {
-                BOOST_THROW_EXCEPTION(InvalidConfigurationException(
-                    "Phase Config 'Threads' parameter is not supported if "
```

Comment:
What was the original rationale for this? Does it shed any light on whether there's a reason for both phase and actor level config?

Reply:
@jimoleary this was your code I think

Reply:
This code ensures that the multiple threads code doesn't use the phase threads configuration as it doesn't really need it.

I think that if we were going forward with this version of the code we should throw an error when there is phase thread configured. Although I am not sure if we have covered all the cases especially w.r.t some of the workloads